apps/api/src/app/integrations/usecases/update-integration/update-integration.usecase.ts

```diff
@@ -20,7 +21,9 @@ export class UpdateIntegration {
   async execute(command: UpdateIntegrationCommand): Promise<IntegrationEntity> {
     const existingIntegration = await this.integrationRepository.findById(command.integrationId);
     if (!existingIntegration) throw new NotFoundException(`Entity with id ${command.integrationId} not found`);
-
+    if (NOVU_START_PROVIDERS.includes(existingIntegration.providerId)) {
+      command.limits = NOVU_START_PROVIDERS_LIMITS;
```

Comment:
This needs to change when we add another novu provider because it can be different limits for different providers.